### ISSUE TITLE
Can't import with maven

### ISSUE THREAD

**Issue by sanslash332** [bedeho/bencodej#1]
Hello, I'm trying to  import the lib to my Project simply puting the code that you put in the readme, but eclipse can't resolve the dependency.
Is missing in the readme some repository that I have to add to the pon.xml to can import it?

I have to clone the repo, import to a eclipse Project and with that use in my Project.

Thanks for all.

**Comment by bedeho** [comment]
I don't believe I actually was able to get it published in a Maven repo, that is probably the reason.

**Comment by sanslash332** [comment]
Sure, that is the problem :P 

 

For use it, I have to import it into eclipse, and re generate the project :P
